Makefile

```diff
@@ -29,6 +29,7 @@ setup:
 	opam install --deps-only --with-test ./vendor/ocamlformat-0.16.0/ocamlformat_lib.opam
 	opam install --deps-only --with-test ./vendor/ocaml-lsp-1.4.0/jsonrpc.opam
 	opam install --deps-only --with-test ./vendor/ocaml-lsp-1.4.0/lsp.opam
+	opam install ocamlformat=0.16.0  # this is required, since ocaml-lsp pulls in ocamlformat 0.17
```

Comment:
```suggestion
```

Hey @erszcz
I managed to reproduce this issue in my environment after doing `opam update` and `opam upgrade`.

I tried out your fix and this line was not necessary. The other two updates to `caramel.opam` and `dune-project` files were sufficient. Can you try removing this one line to see if it works for you locally and here on CI?